test/circuit_breaker_test.rb

```diff
@@ -403,19 +403,53 @@ def test_success_count_on_error_rate_calculation
       assert notifier.notified?, 'no notification sent'
     end
 
+    def test_not_notify_circuit_execution_time_when_not_specified
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier)
+      circuit.run { 'success' }
+      refute notifier.metric_sent?, 'execution time metric sent'
+    end
+
+    def test_send_execution_time_metric
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier, notify_circuit_execution_time: true)
+      circuit.run { 'success' }
+      assert notifier.metric_sent?, 'no execution time metric sent'
+    end
+
+    def test_no_execution_time_metric_on_error_execution
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier, notify_circuit_execution_time: true)
+      circuit.run { raise Timeout::Error }
+      refute notifier.metric_sent?, 'execution time metric sent'
+    end
+
+    def test_no_execution_time_metric_when_circuit_open
+      notifier = gimme_notifier(metric: :execution_time, metric_value: Gimme::Matchers::Anything.new)
+      circuit = Circuitbox::CircuitBreaker.new(:yammer, notifier_class: notifier, notify_circuit_execution_time: true)
+      circuit.send(:open!)
```

Comment:
What if we stubbed ```open?```? I mostly want to see what your thoughts are on that vs what we're currently doing here.